### PR TITLE
Create mod.xml

### PR DESCRIPTION
**Pull request by Digixu**
Adding Hit-J to catalog

### CODE REVIEW

File: mods/[Tsunamods] Hit-J/mod.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  <ID>6728d336-9cb6-4e97-8ab7-474f435e1f6d</ID>
+    <Name>[Tsunamods] Hit-J</Name>
+    <Author>Digixu</Author>
+    <Category>Battle Textures</Category>
+    <Description>4x AI Upscale of battle effects - Hits/Particles/Status Effects/Dust/Enemy Abilities
+
+magic to come soon! </Description>
+    <LatestVersion>
+      <Link>iroj://GDrive/1nqrnRmkvgAgp40DMzRkz5CkZ7iwLTyeB</Link>
+      <Version>V1.0</Version>
```
Comment (julianxhokaxhiu) [inline mods/[Tsunamods] Hit-J/mod.xml:12]: Please remove the `V`

**Review by julianxhokaxhiu — CHANGES_REQUESTED** [review]
Very minor change requires, but we're almost there :) Also looking forward to this one!

File: mods/[Tsunamods] Hit-J/mod.xml (hunk 1 above)
Comment (Digixu) [reply mods/[Tsunamods] Hit-J/mod.xml]: Done, and thanks

**Review by julianxhokaxhiu — APPROVED** [review]

**Comment by julianxhokaxhiu** [comment]
Thank you!

**Pull request merged**